app/src/main/java/net/kuama/documentscanner/viewmodels/CropperModel.kt

```diff
@@ -21,6 +21,7 @@ class CropperModel : ViewModel() {
     val corners = MutableLiveData<Corners>()
     val originalBitmap = MutableLiveData<Bitmap>()
     val bitmapToCrop = MutableLiveData<Bitmap>()
+    val errors = MutableLiveData<Throwable>()
```

Comment:
```suggestion
    private val _errors = MutableLiveData<Throwable>()
    val errors: LiveData<Throwable>
        get = _errors
```
I will evaluate to use the pattern public LiveData- private MutableLiveData so that from external you can't modify the data contained inside this val.
Maybe in a refactor pr not on this one

Reply:
I aggree